arviz/data/base.py

```diff
@@ -266,7 +309,7 @@ def dict_to_dataset(
     index_origin=None,
     skip_event_dims=None,
 ):
-    """Convert a dictionary of numpy arrays to an xarray.Dataset.
+    """Convert a pytree of numpy arrays to an xarray.Dataset.
```

Comment:
```suggestion
    """Convert a dictionary or pytree of numpy arrays to an xarray.Dataset.
```

Reply:
All dictionaries are pytrees, only some pytrees are dictionaries. I added a little more text to help -- let me know what you think.